client/src/components/ItemSummary.jsx

```diff
@@ -41,8 +50,21 @@ class ItemSummary extends React.Component {
         <Description
           description={this.state.item.length ? this.state.item[0].description : null}
         />
-        <ColorSelector colors={this.state.item.length ? JSON.parse(this.state.item[0].colors) : null}/>
-        <Accordions />
+        <ColorSelector
+          colors={this.state.item.length ? JSON.parse(this.state.item[0].colors) : []}
+          selectColor={this.selectColor}
+          selectedColor={this.state.selectedColor}
+        />
+        <SizeSelector
+          sizes={this.state.item.length ? JSON.parse(this.state.item[0].sizes) : []}
+        />
+        <ShippingMock />
+        <Accordion
+          fit={this.state.item.length ? this.state.item[0].fit : null}
```

Comment:
This feels refactorable, but you could also assign `this.state.item.length` to its own variable in the render function.

Reply:
Agreed, I don't like the look of it but I have lots of other things to work on atm. It feels hacky but it's rendering what I want :)